### DISCUSSION TITLE
Load shedding for DGS/ DGS + Webflux

### DISCUSSION THREAD

**Discussion by saravind-hotstar** [Netflix/dgs-framework#1539] [Q&A]
As part of resiliency in DGS framework, is there recommendation on following 
- Preferred library for load shedding with GQL constructs support of query 
- Is https://github.com/Netflix/concurrency-limits preferred to be integrated for DGS webflux considering reactive nature

**Comment by srinivasankavitha** [comment]
Hi @saravind-hotstar - The framework does not come with any support for load-shedding out of the box. Since this is based on spring boot, you can add additional filters for throttling/concurrency limits. We don't recommend anything specific as it depends on the needs of each type of application. Internally , some applications do use the concurrency-limits library but for the MVC stack. We do not use webflux stack internally either and don't plan to prioritize new features for the same in the near future at least.
